src/rocketsRedux/rocketsSlice.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  rockets: [],
+};
+
+const rocketsSlice = createSlice({
+  name: 'rockets',
+  initialState,
+  reducers: {
+    addData(state, action) {
+      // eslint-disable-next-line no-param-reassign
+      state.rockets = action.payload.map((rocket) => ({
+        id: rocket.id,
+        name: rocket.rocket_name,
+        description: rocket.description,
+        image: rocket.flickr_images,
+      }));
+    },
+  },
+});
+
+export const rocketsActions = rocketsSlice.actions;
```

Comment:
- [OPTIONAL] this way of exporting the action is right and functional, however it could be written in a more concise and elegant way using the object destructuring method for ex: `export const { addData } = rocketsSlice.actions;` and then you can dispatch like this `dispatch(addData(data))`

Reply:
Thank  you, I will consider implementing it.